README.md

```diff
@@ -0,0 +1,108 @@
+# Perform
+
+Easy dependency injection for storyboard segues.
+
+```swift
+import Perform
+
+// ...
+
+func tableView(_ tableView: UITableView, didSelectRowAt indexPath: NSIndexPath) {
+  let task = taskList[indexPath.row]
+  perform(.showTaskDetails) {
+    $0.task = task
```

Comment:
What I'm really going for here is to show how concise this can be. I tried adding an explaining comment but I felt like it made the example less punchy. What do you think about this?

```swift
  perform(.showTaskDetails) { taskVC in
    taskVC.task = task
  }
```

Certainly makes it plain what type we're dealing with. In practice I feel like I want to encourage the placeholder style, though. Do you feel there's a readability problem with that style?